src/seedu/addressbook/util/FilePathUtil.java

```diff
@@ -0,0 +1,50 @@
+package seedu.addressbook.util;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * This is a utility class for file path validation.
+ */
+public class FilePathUtil {
```

Comment:
No, there should not be inner classes either. Just put back the methods in AddressBook class (just like you had at the very beginning)